src/backoff.erl

```diff
@@ -52,16 +72,30 @@ fire(#backoff{current=Delay, value=Value, dest=Dest}) ->
 -spec get(backoff()) -> pos_integer().
 get(#backoff{current=Delay}) -> Delay.
 
+%% Toggles jitter feature.
+-spec jitter(backoff()) -> backoff().
+jitter(B=#backoff{jitter=false}) ->
```

Comment:
Actually I've seen the usage by re-reading your edited README. Wondering if we could instead have a function like `jitter(B, on)` `jitter(B, false)` or rather `type(B, jitter)` and `type(B, normal)` to let us swap back and forth between states in an idempotent manner.